doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -469,10 +456,19 @@ arguments used for configuration.  Here is a table describing these emulated dev
 
    * - ``virtio-net``
      - Virtio network type device, parameter should be appended with the format:
-       ``virtio-net,<device_name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>]``.
-       The ``mac`` address is optional, ``device_name`` is the name of the TAP (or MacVTap) device.
-       It must include the keyword ``tap``.
+       ``virtio-net,<device_type=name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>],[mac_seed=<seed_string>]``.
```

Comment:
```suggestion
       ``virtio-net,<device_type>=<name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>],[mac_seed=<seed_string>]``.
```

Reply:
As previous.
Is better to follow the current format?

Reply:
Not following you, but this is wrong.

Reply:
Still wrong...